commands.go

```diff
@@ -125,9 +125,10 @@ type dropMeasurementCommand struct {
 }
 
 type createMeasurementSubcommand struct {
-	Name   string              `json:"name"`
-	Tags   []map[string]string `json:"tags"`
-	Fields []*Field            `json:"fields"`
+	Name          string              `json:"name"`
+	Tags          []map[string]string `json:"tags"`
+	MarshaledTags []string            `json:"marshaledTags"`
```

Comment:
No need to export this, so perhaps make it `marshaledTags` so it's clear to the next guy.